angular/projects/spark-angular/src/lib/components/sprk-accordion-item/sprk-accordion-item.component.ts

```diff
@@ -19,106 +21,145 @@ import { toggleAnimations } from '../sprk-toggle/sprk-toggle-animations';
   /**
-   * The value supplied will be assigned
-   * to the `data-id` attribute on the
-   * component. This is intended to be
-   * used as a selector for automated
-   * tools. This value should be unique
-   * per page.
+   * The value supplied will be assigned to the `data-id` attribute on the
+   * component. This is intended to be used as a selector for automated tools.
+   * This value should be unique per page.
    */
   @Input()
   idString: string;
   /**
-   * Expects a space separated string
-   * of classes to be added to the
-   * component.
+   * Expects a space separated string of classes to be added to the component.
    */
   @Input()
   additionalClasses: string;
+  // TODO - Remove as part of Issue 3597
   /**
-   * Expects a space separated string
-   * of classes to be added to the
-   * heading in the Accordion item.
+   * Deprecated - use `headingAdditionalClasses` instead. Expects a space
+   * separated string of classes to be added to the heading in the Accordion
+   * Item.
    */
   @Input()
   additionalHeadingClasses: string;
   /**
-   * The Accordion item will use this to decide
-   * if it should be open or closed on first render.
-   * (Interacting with the toggle will override this input.)
+   * Expects a space separated string of classes to be added to the heading in
+   * the Accordion Item.
+   */
+  @Input()
+  headingAdditionalClasses: string;
+  /**
+   * The Accordion item will use this to decide if it should be open or closed
+   * on first render. (Interacting with the toggle will override this input.)
    */
   @Input()
   isOpen = false;
+  // TODO - Remove as part of Issue 3597
   /**
-   * If `true`, the active CSS class
+   * Deprecated - use additionalClasses instead. If `true`, the active class
    * will be applied to the item.
    */
   @Input()
   isActive: boolean;
+  // TODO - Remove this input and move the default value to iconNameClosed
+  // as part of issue 3597
   /**
-   * The name of the icon to use for
-   * a closed Accordion item.
+   * Deprecated - use `iconNameClosed` instead. The name of the icon to use for
+   * a closed Accordion Item.
    */
   @Input()
   iconTypeClosed = 'chevron-up-circle';
   /**
-   * The name of the icon to use for
-   * an open Accordion item.
+   * The name of the icon to use for a closed Accordion Item.
+   */
+  @Input()
+  iconNameClosed;
+  // TODO - Remove this input and move the default value to iconNameOpen
+  // as part of issue 3597
+  /**
+   * Deprecated - use `iconNameOpen` instead. The name of the icon to use for
+   * an open Accordion Item.
    */
   @Input()
   iconTypeOpen = 'chevron-up-circle';
   /**
-   * The name of the icon to use before
-   * the title in the Accordion item.
+   * The name of the icon to use for an open Accordion Item.
+   */
+  @Input()
+  iconNameOpen;
+  /**
+   * The name of the icon to use before the heading in the Accordion Item.
    * This is optional.
    */
   @Input()
   leadingIcon: string;
```

Comment:
I know it wasn't part of the AC, but let's add an input for `leadingIconAdditionalClasses` to keep the API in sync with React.